.github/stale.yml

```diff
@@ -0,0 +1,32 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Partner"
+  - "React Core Team"
+  - "Resolution: Backlog"
+  - "Type: Bug"
+  - "Type: Discussion"
+  - "Type: Regression"
+# Label to use when marking an issue as stale
+staleLabel: "Resolution: Stale"
+issues:
+  # Comment to post when marking an issue as stale.
+  markComment: >
+    This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.
+    You may also label this issue as "Type: Bug", "Type: Discussion", or "Resolution: Backlog" and it will remain open.
```

Comment:
This line will likely confuse people, since only contributors can add labels to issues.